ci-operator/step-registry/baremetalds/ipi/pre/baremetalds-ipi-pre-chain.yaml

```diff
@@ -5,6 +5,9 @@ chain:
   - ref: ipi-install-rbac
   - ref: baremetalds-devscripts-proxy
   - ref: baremetalds-devscripts-setup
+  env:
+  - name: PACKET_PLAN
+    default: "n2.xlarge.x86"
```

Comment:
maybe I missed the relevant discussion, but is there a good reason to have a default for the step and a different one for the chain?

Reply:
Yeah, I was asked to not change the default in the step to reduce the number of jobs affected by the change.